webapp/src/components/Description.tsx

```diff
@@ -0,0 +1,30 @@
+import LinkIcon from "@mui/icons-material/InsertLink";
```

Comment:
As discussed offline, to be more semantically rigorous, you can use the identical `Link` icon (instead of `InsertLink`). That way, if ever one day Material MUI changes the `InsertLink` to add a + sign or something, we won't be affected.
```suggestion
import LinkIcon from "@mui/icons-material/Link";
```